app/code/core/Mage/Adminhtml/Block/System/Config/Form.php

```diff
@@ -423,6 +424,9 @@ public function initFields($fieldset, $group, $section, $fieldPrefix = '', $labe
                     }
 
                     $sourceModel = Mage::getSingleton($factoryName);
+                    if (!$sourceModel) {
+                        Mage::throwException("Entity type model '{$factoryName}' is not found");
```

Comment:
```suggestion
                        Mage::throwException("Source model '{$factoryName}' is not found");
```
@elidrissidev right, I kinda tried to use the same message everywhere for translation porpuses  but these messages aren't translated anyway.. fixed :-)